src/main/cpp/autos/ScorePreloadedAuto.cpp

```diff
@@ -17,18 +18,18 @@ ScorePreloadedAuto::ScorePreloadedAuto(std::shared_ptr<SwerveDrive> swerve, std:
     double negative_pickup_degrees = RobotParams::GetParam("arm.negative_pickup.arm_setpoint", 0.0);
     double negative_pickup_extend = RobotParams::GetParam("arm.negative_pickup.extend_setpoint", 1.0);
 
+    double score_position = RobotParams::GetParam("arm.score_high.intake_setpoint", 0.0);
+
     AddCommands(
         IntakeCommand(intake, -0.1), // hold the cube/cone in
         ArmSetStateCommand(arm, TeamOKC::ArmState(extend, degrees)),
-        frc2::WaitCommand(units::second_t(4.5)), // wait for the command to finish
+        frc2::WaitCommand(units::second_t(3.0)), // wait for the command to finish
+        frc2::WaitCommand(units::second_t(1.5)), // wait for the command to finish
```

Comment:
Non blocking comment (let's get this merged) but in a future PR this should be a parameter

Reply:
I guess that may require scripted auto, so you don't need to prioritize this request